src/database/queries/entity.js

```diff
@@ -0,0 +1,64 @@
+const Entity = require('../models/index').Entity
+module.exports = class UserEntityData {
+	static async createEntity(data) {
+		try {
+			return await Entity.create(data, { returning: true })
+		} catch (error) {
+			console.log(error)
+			throw error
+		}
+	}
+
+	static async findAllEntities(filter) {
+		try {
+			filter.status = 'ACTIVE'
+			return await Entity.findAll({ where: filter })
+		} catch (error) {
+			throw error
+		}
+	}
+
+	static async updateOneEntity(id, update, options = {}) {
+		try {
+			const filter = {
+				id: id,
+			}
+			const [rowsAffected] = await Entity.update(update, {
+				where: filter,
+				...options,
+			})
+
+			if (rowsAffected > 0) {
+				return 'ENTITY_UPDATED'
+			} else {
+				return 'ENTITY_NOT_FOUND'
+			}
+		} catch (error) {
+			throw error
+		}
+	}
+
+	static async deleteOneEntityType(id, userId) {
+		try {
+			const rowsAffected = await Entity.destroy({
+				where: {
+					id: id,
+					created_by: userId,
+				},
+			})
+
+			return rowsAffected > 0 ? 'ENTITY_UPDATED' : 'ENTITY_NOT_FOUND'
```

Comment:
We are not going ahead with this pattern right? Queries should return the raw responses of the DB engine or ORM, and let he service layer deal with the interpretation of what it means. 

Unless the interpretation itself is very obvious. [DISCUSS]

Reply:
resolved [here](https://github.com/ELEVATE-Project/mentoring/pull/258/files#diff-6de8a4314ea72d104613372fdbc910ffec808545d44a776c725158d0ba779734).

Reply:
resolved [here](https://github.com/ELEVATE-Project/mentoring/pull/258/files#diff-5bbe22b9ee87453d12b0da1f2ca95408e27b69df6566fb1570bee928946c3ce7)